pkgs/tools/backup/dedup/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchurl, lz4, snappy, openmp ? null }:
+
+stdenv.mkDerivation rec {
+  pname = "dedup";
+  version = "1.0";
+
+  src = fetchurl {
+    url = "https://dl.2f30.org/releases/${pname}-${version}.tar.gz";
+    sha256 = "0wd4cnzhqk8l7byp1y16slma6r3i1qglwicwmxirhwdy1m7j5ijy";
+  };
+
+  makeFlags = [
+    "CC:=$(CC)"
+    "PREFIX=${placeholder "out"}"
+    "MANPREFIX=${placeholder "out"}/share/man"
+  ] ++ stdenv.lib.optional (openmp != null) [
```

Comment:
I'm not sure I understand the value of making openmp optional if you unconditionally pass it in in `all-packages.nix`.